.github/workflows/spec-gen-sdk-status.yml

```diff
@@ -0,0 +1,33 @@
+name: "[TEST IGNORE] spec-gen-sdk status"
```

Comment:
I've noticed that this check is failing in Event Grid specs for a few languages. But the title is [TEST IGNORE]. Are failures expected here?

Reply:
You could ignore this check status as this is a newly added check which is in testing phase. However, I noticed event grid typespecs fails in the TSV validation. Do you know what's the issue?

Reply:
This should be fixed if you sync your branch from main. But regarding these statuses, do they block regular merging?

Reply:
I also wonder if they could show a warning status instead of a failure if they are still in testing phase. Ideally, we would be able to know if a PR can be merged or not based on the CI checks - otherwise we end up with alert fatigue and can miss errors that we shouldn't be ignoring (as in the case you mention with the TSV validation).

Reply:
These do not block merging as they are not required checks. Please just ensure the required checks are passing and watch the "Next Steps to Merge" comment for what needs to be done.